.github/workflows/build.yaml

```diff
@@ -30,3 +30,48 @@ jobs:
       run: make test
     - name: Verify
       run: make verify
+
+  kind-linux-arm64:
+    # Hosted on Equinix
+    # Docs: https://github.com/fluxcd/flux2/tree/main/.github/runners
+    runs-on: [self-hosted, Linux, ARM64, equinix]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
+      - name: Run tests
+        run: |
+          mkdir tmp-download; cd tmp-download; go mod init go-download;
+          GOBIN="${GITHUB_WORKSPACE}/build/gobin" go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+          GOBIN="${GITHUB_WORKSPACE}/build/gobin" go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0
+          GOBIN="${GITHUB_WORKSPACE}/build/gobin" go install github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0
```

Comment:
When building against the arm64 runners for SC, only the `setup-envtest` did not work via Makefile. Did you also have issues with `controller-gen` and `gen-crd-api-reference-docs`?

Reply:
yes: https://github.com/fluxcd/image-automation-controller/runs/5110912422?check_suite_focus=true

Reply:
Ca we set `GOBIN` for Make and get rid of the duplicate install? The issue with this is that the controller-gen version will diverge from the one in the Makefile.